zad2/warriors.rb

```diff
@@ -1,12 +1,181 @@
-class Warrior
-  def initialize(name:, level:)
+require 'faker'
+# Error that has to be raised if the wrong type of characters are entering the BattleArena (non-Character class)
+class ArenaEntryError < StandardError
+  def message
+    'You probably try to make some integers fight each other, you sadistic bastard! Check if the competitors are Characters before inviting them to the Arena.'
```

Comment:
można?
https://www.youtube.com/watch?v=-y6UgJXocfs
można!

Reply:
O, rzeczywiście, o tym nie pomyślałem - całe życie się człowiek uczy :) Choć najwyraźniej z uwagi na okrucieństwo liczb na polu walki autor materiału skorzystał z pomocy aktorów. 
Na deser: 19 and 20 got into a fight. 21.